pkg/api/v1alpha1/thirdparty/tinkerbell/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
Curious why this is changing

What commands did you run?
What go version are you using?

Reply:
we shouldn't gofumpt autogenerated files, could we remove it?

Reply:
make was failing and asking me to reformat and then it reformatted all these files. let me remove them.